website/content/v1.10/kubernetes-guides/configuration/etcd-metrics.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Expose the Etcd Metrics Endpoint"
+description: "Learn how to the expose etcd metrics endpoint."
+---
+
+To allow monitoring tools to collect metrics from your etcd database, you need to explicitly expose the etcd metrics endpoint.
+
+Here's how to do it:
+
+1. Create a patch file named `etcd-metrics-patch.yaml` that exposes the etcd metrics endpoint on `port:2381`, accessible from all network interfaces
+
+    ```shell
+    cat << EOF > etcd-metrics-patch.yaml
+    - op: add
+    path: /cluster/etcd/extraArgs
+    value:
+        listen-metrics-urls: http://0.0.0.0:2381
+    EOF
+    ```
+
+1. Create a `CP_IPS` variable that contains the IP addresses of your control plane nodes:
+
+    ```bash
+    CP_IPS="<control-plane-ip-1>,<control-plane-ip-2>,<control-plane-ip-3>"
+    ```
+
+   **Note**: Don’t use a load balancer or VIP endpoint for the control-plane-ips.
```

Comment:
this is a fine note, but I don't think it belongs here. In fact people don't use VIP for `--nodes`, they might use VIP for `--endpoints` (both are bad)